orga/app/components/campaign/settings/view.hbs

```diff
@@ -5,7 +5,7 @@
         <dt class="label-text campaign-settings-content__label">{{t "pages.campaign-settings.campaign-type.title"}}</dt>
         <dd class="content-text campaign-settings-content__text">{{this.campaignType}}</dd>
       </div>
-      {{#if @campaign.isTypeProfilesCollection}}
+      {{#if this.multipleSendingEnable}}
```

Comment:
**suggestion** : Pour le variable booléennes on aime bien préfixé avec un verbe. Ex : is, has, are, ...